tests/tracing/export/test_mlflow_v3_exporter.py

```diff
@@ -39,37 +36,26 @@ def _flush_async_logging():
 
 
 @pytest.mark.parametrize("is_async", [True, False], ids=["async", "sync"])
-@pytest.mark.parametrize("experiment_id", [None, _EXPERIMENT_ID])
-def test_export(experiment_id, is_async, monkeypatch):
+def test_export(is_async, monkeypatch):
     monkeypatch.setenv("DATABRICKS_HOST", "dummy-host")
     monkeypatch.setenv("DATABRICKS_TOKEN", "dummy-token")
     monkeypatch.setenv("MLFLOW_ENABLE_ASYNC_TRACE_LOGGING", str(is_async))
 
-    mlflow.tracing.set_destination(Databricks(experiment_id=experiment_id))
+    mlflow.tracing.set_destination(Databricks(experiment_id=_EXPERIMENT_ID))
 
-    # Create mock for returned trace from _start_trace_v3
-    mock_trace_info = TraceInfo(
-        trace_id="12345",
-        trace_location=TraceLocation(
-            type=TraceLocationType.MLFLOW_EXPERIMENT,
-            mlflow_experiment=MlflowExperimentLocation(experiment_id=_EXPERIMENT_ID),
-        ),
-        request_time=1234567890,
-        state=TraceState.OK,
-        request_preview="Some request",
-        response_preview="Some response",
-        client_request_id=None,
-        execution_duration=100,
-        trace_metadata={"key1": "value1"},
-        tags={"foo": "bar"},
-    ).to_proto()
-    mock_response = pb.StartTraceV3.Response(
-        trace=pb.Trace(trace_info=mock_trace_info),
-    )
```

Comment:
This mock trace is inconsistent with the actual trace exported by the client, resulting in a size mismatch in the test case. To fix this, I grabbed a reference to the actual trace using a `mock_response` function.